### PR TITLE
Do not require the mock package for python 3

### PR DESCRIPTION
**Pull request by mpessas**

### CODE REVIEW

File: freezefrog/__init__.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import calendar
 import datetime
-from mock import patch
+
+try:
+    from mock import patch
+except ImportError:
+    from unittest.mock import patch
```
Comment (thomasst) [inline freezefrog/__init__.py:5]: Should this go the other way round? I.e. give priority to the built-in?

File: .circleci/config.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ defaults: &defaults
   - checkout
   - run:
       name: Install dependencies
-      command: sudo pip install -r requirements.txt flake8 pytest
```
Comment (thomasst) [inline .circleci/config.yml:16]: Do we need (make any changes to) requirements.txt?

Comment (mpessas) [reply .circleci/config.yml:16]: Actually, I will delete the requirements.txt file.

File: freezefrog/__init__.py (hunk 1 above)
Comment (mpessas) [reply freezefrog/__init__.py:5]: Sure

File: setup.py [hunk 3]
```diff
@@ -11,10 +11,9 @@
     maintainer_email='engineering@close.io',
     description='Efficient datetime mocking in tests',
     test_suite='tests',
-    tests_require=['mock'],
     platforms='any',
     install_requires=[
-        'mock',
+        'mock==2.0.0;python_version<"3"',
```
Comment (thomasst) [inline setup.py]: We should not pin `mock` to a specific version here. `mock>=2` could be fine though.

> install_requires is a setuptools setup.py keyword that should be used to specify what a project **minimally** needs to run correctly

> **It is not considered best practice to use install_requires to pin dependencies to specific versions**, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

 – https://packaging.python.org/discussions/install-requires-vs-requirements/

**Review by thomasst — APPROVED** [review]

**Pull request merged**